src/ioctl/mod.rs

```diff
@@ -301,19 +282,19 @@ pub enum Direction {
 pub type IoctlOutput = c::c_int;
 
 /// The type used by the `ioctl` to signify the command.
-pub type RawOpcode = _RawOpcode;
+pub type Opcode = _Opcode;
```

Comment:
Sorry for being late, but would a newtype struct be better than a type alias? That will be more informative in IDE and can avoids accidental mixing integers in.
Not sure if it interferes with const eval though.

Reply:
Newtypes are incompatible with const generics.